grammar.js

```diff
@@ -1779,6 +1781,18 @@ module.exports = grammar({
           field('right', $._expression)
         ))
       ),
+      ...[
+        [$.keyword_between, 'binary_between'],
+        [$.not_between, 'binary_between'],
+      ].map(([operator, precendence]) =>
+        prec.left(precendence, seq(
+          field('left', $.identifier),
+          field('operator', operator),
+          field('low', $.literal),
+          $.keyword_and,
+          field('high', $.literal)
+        ))
```

Comment:
I'm not sure if this answers your question, I can look into it more later but from looking at this I don't think `between` & `not between` should be `binary_expression` nodes. They might deserve their own nodes.

Reply:
I have made it an extra node. However, as soon as the `not` token in in there it fails because it detects the where clause as a `binary_expression`